darwin/hax_driver/com_intel_hax/intelhaxm.xcodeproj/project.pbxproj

```diff
@@ -73,8 +73,23 @@
 		CFB6FDDB1ED43C540048A750 /* ramblock.c in Sources */ = {isa = PBXBuildFile; fileRef = CFB6FDDA1ED43C540048A750 /* ramblock.c */; };
 		CFD697471ED2DC9700F10631 /* gpa_space.c in Sources */ = {isa = PBXBuildFile; fileRef = CFD697461ED2DC9700F10631 /* gpa_space.c */; };
 		CFD697491ED2DCB700F10631 /* memory.h in Headers */ = {isa = PBXBuildFile; fileRef = CFD697481ED2DCB700F10631 /* memory.h */; };
+		FA8F651E208BAD9A00C8E91F /* emulate.c in Sources */ = {isa = PBXBuildFile; fileRef = FA8F651D208BAD9A00C8E91F /* emulate.c */; };
+		FA8F6520208BC1BA00C8E91F /* emulate_ops.asm in Sources */ = {isa = PBXBuildFile; fileRef = FA8F651F208BC1BA00C8E91F /* emulate_ops.asm */; };
 /* End PBXBuildFile section */
 
+/* Begin PBXBuildRule section */
+		FA8F6523208BDF6600C8E91F /* PBXBuildRule */ = {
+			isa = PBXBuildRule;
+			compilerSpec = com.apple.compilers.proxy.script;
+			fileType = sourcecode.nasm;
+			isEditable = 1;
+			outputFiles = (
+				"$(DERIVED_FILE_DIR)/${INPUT_FILE_BASE}.o",
+			);
+			script = "/usr/local/bin/nasm -f macho64 ${INPUT_FILE_PATH} -o ${SCRIPT_OUTPUT_FILE_0}\n";
```

Comment:
1. Add `--prefix _` to generate symbol names with a leading underscore, so as to avoid symbol resolution errors when we load `intelhaxm.kext`. ([Reference](https://github.com/jonof/jfbuild/issues/3))
1. nit: Move the required parameter (input file) to the end.

In summary:

```
script = "/usr/local/bin/nasm -f macho64 --prefix _ -o ${SCRIPT_OUTPUT_FILE_0} ${INPUT_FILE_PATH}\n";
```